website/server/models/user/hooks.js

```diff
@@ -13,6 +13,7 @@ schema.plugin(baseModel, {
   private: ['auth.local.hashed_password', 'auth.local.salt', '_cronSignature'],
   toJSONTransform: function userToJSON (plainObj, originalDoc) {
     plainObj._tmp = originalDoc._tmp; // be sure to send down drop notifs
+    plainObj.filters = undefined;
```

Comment:
Not really because some users may have an empty object saved and while mongoose doesn't save new data that is not in the database it keeps anything that is in mongo but not in the model